rotkehlchen/tests/api/test_current_assets_price.py

```diff
@@ -261,6 +266,8 @@ def test_remove_manual_current_price(rotkehlchen_api_server):
     )
     # usd manual current price should have not been touched
     assert GlobalDBHandler().get_manual_current_price(A_USD) == (A_EUR, Price(FVal(25)))
+    # Check that the cache in the inquierer has been invalidated
+    assert (A_ETH, A_EUR) not in Inquirer()._cached_current_price
```

Comment:
I think if you add other possible pairs e.g `(A_ETH, A_USD)`, you'd find that they were not invalidated

Reply:
the lastest price will get replaced and the old will become historical price